ironfish-graph-explorer/src/index.js

```diff
@@ -37,17 +37,23 @@ function getNodes() {
         const nodeOffsetX = (islandOffset * ISLAND_OFFSET) + (childOffset * CHILD_OFFSET)
         const nodeOffsetY = block.seq * LEVEL_OFFSET
 
+        const graffiti = block.graffiti.replace(/\0/g, '').trim()
```

Comment:
This replaces the null terminator with '', which these empty buffers are filled with